docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -746,13 +746,65 @@ To read and write a value of the Bluetooth descriptor:
       - write about addConnectStateChangeListener()
       - write about removeConnectStateChangeListener()
```

Comment:
This whole TODO comment can be now removed.

Reply:
All TODOs will be removed in the last commit.